airbyte-integrations/connectors/source-hubspot/poetry.lock

```diff
@@ -2,28 +2,31 @@
 
 [[package]]
 name = "airbyte-cdk"
-version = "0.78.6"
+version = "0.87.0"
```

Comment:
Actually, testing manually makes me believe that upping the CDK will cause issues with `ClientSideIncrementalStream` as the state will be set and [this](https://github.com/airbytehq/airbyte/blob/1188221473e2156a591b12c655e6b853e2df65d7/airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py#L862-L864) will fail with an index error. I think it comes from a change [here](https://github.com/airbytehq/airbyte/blob/1188221473e2156a591b12c655e6b853e2df65d7/airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py#L215) where the condition before was `if stream_state and "state" in dir(stream_instance) and <...>` but we've removed the first part